vizro-core/docs/pages/user-guides/layouts.md

```diff
@@ -48,42 +45,51 @@ If that is your desired layout, you can create your charts/components without pr
 
 
 ## Configure the grid
-To customize the grid arrangement, you can configure the `grid` parameter of the [`Layout`][vizro.models.Layout] model.
-The example below shows how the grid works and how to specify a valid one:
+To customize the grid arrangement, configure the `grid` parameter of the [`Layout`][vizro.models.Layout] model.
 
-```python title="Basic Example"
+The example below shows an example of a valid `grid`:
+
+```python title="Basic example"
 grid = [[0, 1],
         [0, 2]]
 ```
 
-- The `grid` needs to be provided as `List[List[int]]` (for example, `grid = [[0, 1], [0, 2]]`)
-- The integers in the `grid` correspond to the index of the chart/component inside the list of `components` provided to [`Page`][vizro.models.Page]
-- The number of integers in the `grid` needs to match the number of chart/components provided
-- Each sub-list corresponds to a grid row (for example, row 1 = `[0, 1]` and row 2 = `[0, 2]`)
+- The `grid` must be provided as `List[List[int]]` (for example, `grid = [[0, 1], [0, 2]]`).
+- The integers in the `grid` must be consecutive integers starting with 0 (for example, `0`, `1`, `2`).
+    - The integers correspond to the index of the chart/component inside the list of `components` provided to [`Page`][vizro.models.Page].
+    - The number of integers in the `grid` needs to match the number of chart/components provided.
+- Each sub-list corresponds to a grid row (in the example above, row 1 = `[0, 1]` and row 2 = `[0, 2]`)
 - Each element inside the sub-list corresponds to a grid column (for example, column 1 = `[0, 0]` and column 2 = `[1, 2]`)
-- The integers in the `grid` need to be consecutive integers starting with 0 (for example, `0`, `1`, `2`)
-- Each chart/component will take the entire space of its grid area (empty spaces are currently not enabled)
-- The area spanned by a chart/component in the grid must be rectangular
-- The grid can be arbitrarily large, allowing arbitrarily granular control of the grid:
-
-```python title="Advanced Example"
-grid=[[0, 1, 3, 4],
-      [2, 2, 3, 4]]
+
+- Each chart/component will take the entire space of its grid area (spaces are not enabled).
+- The area spanned by a chart/component in the grid must be rectangular.
+- The grid can be arbitrarily large, allowing arbitrarily granular control of the grid.
```

Comment:
Spaces are enabled now! The users have to provide -1 to create empty spaces 👍 

```suggestion
- Each chart/component will take the entire space of its grid area.
- The area spanned by a chart/component in the grid must be rectangular.
- The grid can be arbitrarily large, allowing arbitrarily granular control of the grid.
- Empty spaces can be added via providing `-1` in the allocated grid space
```

Reply:
That confused me later on the page, and makes sense now you say they're enabled. I'll delete this or, rather, rephrase it.